### PR TITLE
Fix selector for vm ownership for upstream

### PR DESCRIPTION
**Pull request by otsuman**
{{pytest: cfme/tests/cloud_infra_common/test_vm_ownership.py -k test_form_button_validation --use-provider vsphere55 }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 8c92dfb88a44908f61d8fa771fcc58ac8be47793:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: cfme/common/vm.py [hunk 1]
```diff
@@ -32,8 +32,8 @@
 retire_remove_button = "//span[@id='remove_button']/a/img|//a/img[contains(@src, '/clear')]"
 
 set_ownership_form = Form(fields=[
-    ('user_name', Select("//select[@id='user_name']")),
-    ('group_name', Select("//select[@id='group_name']")),
+    ('user_name', AngularSelect('user_name')),
```
Comment (psav) [inline cfme/common/vm.py]: Does this still work in the old versions, if not, we need to use a verpick

Comment (otsuman) [inline cfme/common/vm.py]: @psav but does we support old versions? I thought for now we supporting 5.6+ and upstream for tests

Comment (psav) [inline cfme/common/vm.py]: That's what I mean, we support 5.6+ (is AngularSelect present in 5.6?)

Comment (otsuman) [inline cfme/common/vm.py]: @psav ci/downstream-56z — success. I think AngularSelect works with 5.5+ versions 
https://github.com/ManageIQ/integration_tests/blob/master/cfme/cloud/provider/__init__.py#L64

Comment (psav) [inline cfme/common/vm.py]: Right I'm merging 5.7 failure is due to another issue

**Pull request merged**